modules/oadp-converting-to-new-dpa-1-2-0.adoc

```diff
@@ -48,4 +48,9 @@ spec:
 # ...
 ----
 
+[NOTE]
+====
+The `fs-backup/restic timeout` is a one-hour default fs-backup/restic timeout. In OADP 1.3.1 the default is four hours.
```

Comment:
oc explain dataprotectionapplications.oadp.openshift.io.spec.configuration.velero.args.fs-backup-timeout

KIND:     DataProtectionApplication
VERSION:  oadp.openshift.io/v1alpha1

FIELD:    fs-backup-timeout <integer>

DESCRIPTION:
     How long pod volume file system backups/restores should be allowed to run
     before timing out. (default 4h0m0s)
     
the default value for dataprotectionapplications.oadp.openshift.io.spec.configuration.velero.args.fs-backup-timeout t is Actually for both restic and kopia (in 1.3.x)